src/app/pages/HomePage/HomePage.tsx

```diff
@@ -41,10 +43,38 @@ export default function HomePage(): JSX.Element {
               iconType={item.iconType}
               name={item.name}
               connections={item.connections}
-              onDeleteClick={() => deleteGear(item.name)}
+              onDeleteClick={() => (
+                setDeleteItem(item), setShowDeleteModal(true)
```

Comment:
as I said earlier, this is not the most common way to call two functions inside one function, but it works. Essentially, the round brackets create a way to return several things, but your function calls don't return anything. It's not bad style, just less common